deep_quoridor/src/agents/greedy.py

```diff
@@ -66,7 +74,7 @@ def _shortest_path_from(self, game: Quoridor, pos: Position, target_row: int) ->
         Given a position in the board, it returns the shortest path to get to the target_row.
         """
         if pos[0] == target_row:
-            return []
+            return [pos]
```

Comment:
If we want the shortest path and start from the target row, we need to include this cell